.github/workflows/continuous-integration.yml

```diff
@@ -672,11 +672,11 @@ jobs:
           BUILDTYPE: vagovprod
           NODE_ENV: production
 
-  testing-reports:
-    name: Testing Reports
+  failure-notifications:
+    name: Failure Notifications
     runs-on: ubuntu-latest
     needs: [cypress-tests-prep, cypress-tests]
```

Comment:
I don't think we need `cypress-tests-prep` in this job. So you can change line 678 to:

`needs: cypress-tests`